### PR TITLE
Adding R kernel packages and dependencies

### PR DESCRIPTION
**Pull request by epifanio**

### CODE REVIEW

**Comment by epifanio** [comment]
Can we merge? [this line](https://github.com/epifanio/OSGeoLive/commit/013695d93ecf0d00dfc1799161ac13431063fa0e#diff-1e6621b3918a19f40b4ea569a0ac9312R86) is still commented inn build 111

**Comment by kalxas** [comment]
I have built several times with this line uncommented, the R kernel is not available in the disk.
Build 111 is for another PR.
We cannot merge until we have a working nightly.
I will provide a new build with your PR included if you need to see the log again.

**Comment by kalxas** [comment]
Build 106 was the one including your changes, but I have deleted it from the servers by mistake.
I am rebuilding it and will let you know here

**Comment by epifanio** [comment]
Thanks, I start to attempt a build locally too .. I'll let you know how it goes.

**Comment by kalxas** [comment]
Builds are very slow the last 48h...

**Comment by kalxas** [comment]
@epifanio build106 is now available based on the commits of this PR

**Comment by kalxas** [comment]
Thanks @epifanio 
Starting a new build in order to check and merge.

**Comment by kalxas** [comment]
On build107 how can I make sure the R kernel is there?
In the Notebook browser UI, only Python2 is listed.

**Comment by epifanio** [comment]
I'm rebuilding and testing with the new commit

File: bin/install_R.sh [hunk 1]
```diff
@@ -83,13 +83,15 @@ apt-get --assume-yes install r-recommended
 # not found in Lucid: r-cran-adapt
 
 #Calls R script to do install with feedback to stdout
-#R --no-save < ../app-conf/R/installRpackages.r
+R --no-save < ../app-conf/R/installRpackages.r
```
Comment (kalxas) [inline bin/install_R.sh]: I propose to remove this line here since it has been correctly moved after the apt-get install

**Comment by kalxas** [comment]
Also what is this .DSfile added?

**Comment by kalxas** [comment]
Looking at the commit history, it seems that you forgot to pull, went back 2 commits, then merged (probably from another pc, your e-mail has changed).
Plus, the commit messages are repeating.
This PR is becoming a history nightmare. We will need to squash in order to merge...
https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/

Another solution would be to do a git reset to 41c41dd8239ab76f3226565c84d5f0b2b9d62b1e and re-apply your changes on top of that

**Comment by epifanio** [comment]
I tried to remove the .DS_store .. are they still there?  is OSX stuff .. it got added when i did a ```git add . ```

**Comment by epifanio** [comment]
my fork was out of sync with master .. and after i resynced, i forgot to sync R branch. Hopefully fixed now.

**Comment by epifanio** [comment]
Seems the failure happen because as default the installer store the kernels in the user home directory, under ``` .local```, last commit should address this issue. (building and testing, I'll report here)

**Comment by kalxas** [comment]
Thanks.
I will try to clean things up a little in the history of this PR once you get this working.

**Comment by epifanio** [comment]
Finally done .. I tested a build with this last commit and it works. @kalxas , please let me know what should I do to clean-up the git history for this pr. Thanks!

**Comment by kalxas** [comment]
Thanks @epifanio will test now and let you know

**Comment by kalxas** [comment]
ok, I have cleaned up the commits here:
https://github.com/kalxas/OSGeoLive/commits/PR66

I am building now based on that branch. Lets see how it goes :)

**Comment by kalxas** [comment]
This is now merged into master.

**Pull request closed without merging**